src/components/CustomButton.js

```diff
@@ -2,15 +2,19 @@ import React from "react";
 import { PseudoBox } from "@chakra-ui/core";
 
 type CustomButtonProps = {
-	variant?: "primary" | "secondary"
+	disabled: bool,
```

Comment:
Again, I believe the data type in JS is "boolean" not "bool" - @AbhinavThukral97 can confirm, due to my unfamiliarity with type module.

Reply:
Done.